enos/modules/vault_verify_write_data/scripts/smoke-enable-secrets-kv.sh

```diff
@@ -33,16 +33,16 @@ test -x "$binpath" || fail "unable to locate vault binary at $binpath"
 retry 5 "$binpath" status > /dev/null 2>&1
 
 # Create user policy
-$binpath policy write reguser -<<EOF
+retry 5 $binpath policy write reguser -<<EOF
 path "*" {
   capabilities = ["read", "list"]
 }
 EOF
 
 # Enable the userpass auth method
-$binpath auth enable userpass > /dev/null 2>&1
+retry 5 $binpath auth enable userpass > /dev/null 2>&1
```

Comment:
Kinda worried about needing retries when using these API's. The first retry makes sense as we might be racing for the server to start up and accept requests. The subsequent retries make me uneasy. Maybe a single retry?